apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -35,14 +35,8 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             isRequired: true,
         },
         
-        feeCalculationUrl: {
-            schemaDoc: 'Url to acquiring integration service. Mobile devices will use it to pass MultiPayment ID and get know about explicitFee. All fees will also be calculated there',
-            type: Text,
-            isRequired: true,
-        },
-
-        paymentUrl: {
-            schemaDoc: 'Url to acquiring integration service. Mobile devices will use it to pass MultiPayment ID and start payment process',
+        hostUrl: {
```

Comment:
May be acquiringIntegrationUrl?

Reply:
Why? We're in `AcquiringIntegration` model already, so if think `AcquiringIntegration` + `hostUrl` already = `acquiringIntegrationUrl` by meaning?